### PR TITLE
fix: <ExpandCollapse> flickering

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/5028746075

Task to revisit animation https://monday.monday.com/boards/3532714909/pulses/5042628555

### CODE REVIEW

File: src/components/Accordion/Accordion/__stories__/accordion.stories.mdx [hunk 1]
```diff
@@ -27,7 +27,9 @@ export const accordionTemplate = args => {
   return (
     <Accordion className="monday-storybook-accordion_small-wrapepr" defaultIndex={[1]} {...args}>
       <AccordionItem title="Notifications">
-        <div className="monday-storybook-accordion_small-box" />
+        <div className="monday-storybook-accordion_small-box">
+          Content
+        </div>
```
Comment (SergeyRoyt) [inline src/components/Accordion/Accordion/__stories__/accordion.stories.mdx:32]: For demo, revert afterwards

File: src/components/ExpandCollapse/ExpandCollapse.module.scss [hunk 2]
```diff
@@ -67,16 +67,17 @@
...
+//@keyframes expandContentAnimation {
+//  0% {
+//    opacity: 0;
+//    transform: translateY(var(--dropdown-menu-fadinY, -75px));
+//  }
+//  100% {
+//    opacity: 1;
+//    transform: translateY(0px);
+//  }
+//}
```
Comment (SergeyRoyt) [inline src/components/ExpandCollapse/ExpandCollapse.module.scss:83]: **_TODO replace animation or remove it_**

File: src/components/ExpandCollapse/ExpandCollapse.module.scss [hunk 3]
```diff
@@ -66,17 +66,18 @@
   transition: transform var(--motion-expressive-short) var(--motion-timing-transition);
 }
 
+// TODO add some other animation here or remove completely - causing flickering cause of intersection with the header
```
Comment (SergeyRoyt) [inline src/components/ExpandCollapse/ExpandCollapse.module.scss:69]: Commented out, will be added to to the backlog

**Review by YossiSaadi — APPROVED** [review]

**Comment by YossiSaadi** [comment]
we should use the `display: grid` hack for that, don't forget to create a task (maybe link it here for future reference)

**Pull request merged**